eric_sse/prefabs.py

```
@@ -75,49 +76,65 @@ def __init__(self, max_workers: int, stream_delay_seconds: int = 0, executor_cla
         self.executor_class = executor_class
 
     async def process_queue(self, listener: MessageQueueListener) -> AsyncIterable[dict]:
+        """Starts a process by creating self.max_workers workers and executing on_message method of listener in parallel."""
 
         with self.executor_class(max_workers=self.max_workers) as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)